README.md

```diff
@@ -1,31 +1,39 @@
-# Go stats collection libraries
+# OpenCensus Libraries for Go
 
 [![Build Status][travis-image]][travis-url] [![GoDoc][godoc-image]][godoc-url]
 
-This is still at a very early stage of development and a lot of the API calls
-are in the process of being changed and might break your code in hte future.
+Opencensus-go is a Go implementation of OpenCensus, a toolkit for
+collecting application performance and behavior monitoring data.
+Currently it consists of three major APIs: tags, stats, and tracing.
+
+This project is still at a very early stage of development and
+a lot of the API calls are in the process of being changed and
+might break your code in hte future.
 
 [travis-image]: https://travis-ci.org/census-instrumentation/opencensus-go.svg?branch=master
 [travis-url]: https://travis-ci.org/census-instrumentation/opencensus-go
 [godoc-image]: https://godoc.org/github.com/census-instrumentation/opencensus-go?status.svg
 [godoc-url]: https://godoc.org/github.com/census-instrumentation/opencensus-go
 
-# Go stats core library
-The Go implementation of opencensus.
+
 TODO: add a link to the language independent opencensus doc when it is available.
 
 ## Installation
 To install this package, you need to install Go and setup your Go workspace on your computer. The simplest way to install the library is to run:
 
 $ go get -u github.com/census-instrumentation/opencensus-go
 
+TODO: convert this to godoc so the above go get command doesn't complain and
+godoc can present this nicely.
+
 ## Prerequisites
 This requires Go 1.8 or later as it uses the convenience function sort.Slice(...) introduced in Go 1.8.
 
-## Tags
+## Tags API
```

Comment:
Should we call this Tags Library?

Reply:
rakyll@ I think here API is better. For me "Tags Library" will imply explanation of the internals of the library (for developers not users of the library) which is not the case here.